README.md

```diff
@@ -1,24 +1,65 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet) ![](<https://img.shields.io/badge/-Ruby-rgb(199%2C%2032%2C%2039)?style=plastic&logo=ruby>) ![](<https://img.shields.io/badge/-Rails-rgb(199%2C%2032%2C%2039)?style=plastic&logo=rails>)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Private Events
 
-Things you may want to cover:
+> In this project, I build a site similar to a private [Eventbrite](https://www.eventbrite.com/) which allows users to create events and then manage user signups. Users can create events and send invitations and parties. Events take place at a specific date and at a location (which you can just store as a string, like “Andy’s House”).
 
-* Ruby version
+> A user can create events. A user can attend many events. An event can be attended by many users. This will require me to model many-to-many relationships and also to be very conscious about my foreign keys and class names.
 
-* System dependencies
+## BUILT WITH
 
-* Configuration
+- Ruby
+- ROR (Active Record Associations)
+- SQLite3
+- Bulma
+- Devise
+- Integration test Rspec & Capybara
+## Getting Started 
 
-* Database creation
+- I've included a gem to improve the ruby coding (you can see how to install it the prerequisites below).
+- I've included linters to improve and cleaning the Ruby code.
 
-* Database initialization
+### Prerequisites
 
-* How to run the test suite
+- Having Ruby (programming language) in your machine. <a href="https://www.ruby-lang.org/en/downloads/">Download Ruby HERE</a>
+- IDE to edit and run the code (I used Visual Studio Code 🔥).
+<a href="https://code.visualstudio.com/Download">Download Visual Studio Code HERE</a>
 
-* Services (job queues, cache servers, search engines, etc.)
+## Install
 
-* Deployment instructions
+To get a local copy up and running follow these simple example steps.
+- Open terminal
+- Clone this project with the command `git clone https://github.com/DanSam5K/Private-Events.git`
+- `cd Private-Events` folder
+- Run `bundle install` in your terminal
+- Run `rails db:migrate` in your terminal
+- Run `rails s` in your terminal to start your local host.
+- In your terminal go to (http://127.0.0.1:3000/) 
+- Sign up for a session and create-events "secrets" or you can just watch them anonymously!
 
-* ...
+Now, you can start interratcing with application
```

Comment:
- [ ] I think you have a typo here. It should be **_interacting_**. I advise you to use Grammarly to avoid such typos

Reply:
Noted!